File: Example/Example/ViewController.m
//
//  ViewController.m
//  Example
//
//  Created by Jonathan Tribouharet
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
    
    self.animatedView.textColor = [UIColor whiteColor];
    self.animatedView.singleUnitWidthForAlignmentCenter = 10;
    self.animatedView.font = [UIFont fontWithName:@"HelveticaNeue-Bold" size:15];
    
    self.animatedView.minLength = 3;
}

- (IBAction)didStartAnimationTouch:(id)sender
{
    [self.animatedView setValue:[NSNumber numberWithInt:(rand() % 5000)]];
    [self.animatedView startAnimation];
}

@end

File: SwiftExample/NumberScrollAnimation/ViewController.swift
//
//  ViewController.swift
//  NumberScrollAnimation
//
//  Created by Jimmy Jose on 18/12/14.
//  Copyright (c) 2014 Varshyl Mobile Pvt. Ltd. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet var animatedView:VMNumberScrollAnimatedView? = VMNumberScrollAnimatedView?()
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        self.animatedView?.textColor = UIColor.whiteColor()
        
        self.animatedView?.font = UIFont(name: "HelveticaNeue-Bold", size: 42)!
        self.animatedView?.minLenght = 3
        
    }

     @IBAction func start(){
    
        self.animatedView?.setValue(NSNumber(int: rand()%5000))
        self.animatedView?.startAnimation()
    
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

